src/gmp/utils/identity.js

```diff
@@ -15,6 +15,9 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
+
+import {parseFloat} from 'gmp/parser';
```

Comment:
Hi, could we avoid this import because it created a cyclic dependency?

Reply:
Yes, you are right. Thanks for pointing it out.